packages/calcite-components/src/components/tile/tile.tsx

```diff
@@ -312,6 +340,8 @@ export class Tile implements InteractiveComponent, SelectableComponent {
           : interactive
             ? "button"
             : undefined;
+    const hasContent = Boolean(description || hasContentEnd || hasContentStart || heading || icon);
```

Comment:
Nitpick: for consistency, can you use `!!` instead? Both are valid, but maybe we should look into having ESLint enforce one vs the other. cc @driskull

Reply:
I agree. `!!` is preferred since its much shorter and easier to read.